ui/oopsyraidsy/data/05-shb/dungeon/dohn_mheg.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+// TODO: Missing Growing tethers on boss 2.
+// (Maybe gather party member names on the previous TIIIIMBEEEEEER cast for comparison?)
+// TODO: Failing to interrupt Dohnfaust Fuath on Watering Wheel casts?
+// (15:........:Dohnfast Fuath:3DAA:Watering Wheel:........:(\y{Name}):)
+
+[{
+  zoneRegex: /^Dohn Mheg$/,
+  damageWarn: {
+    'Dohn Mheg Geyser': '2260', // Water eruptions, boss 1
+    'Dohn Mheg Hydrofall': '22BD', // Ground AoE marker, boss 1
+    'Dohn Mheg Laughing Leap': '2294', // Ground AoE marker, boss 1
+    'Dohn Mheg Swinge': '22CA', // Frontal cone, boss 2
+    'Dohn Mheg Canopy': '3DB0', // Frontal cone, Dohnfaust Rowans throughout instance
+    'Dohn Mheg Pinecone Bomb': '3DB1', // Circular ground AoE marker, Dohnfaust Rowans throughout instance
+    'Dohn Mheg Bile Bombardment': '34EE', // Ground AoE marker, boss 3
+    'Dohn Mheg Corrosive Bile': '34EC', // Frontal cone, boss 3
+    'Dohn Mheg Flailing Tentacles': '3681',
+
+  },
+  triggers: [
+    {
+      id: 'Dohn Mheg Imp Choir',
+      regex: / 1A:........:(\y{Name}) gains the effect of Imp/,
```

Comment:
Can you make these all gainsEffectRegex-style triggers like e2s? You'll need to also add translations for the effect name.  (I can dig it up for you if you need it.)  I think in general, it's also fine to use the effectName where possible for oopsy names, so that no extra translation is needed.

Reply:
To clarify, you would prefer that I add the effects to the respective `lang/$language.js` files? I can do that, but I don't have translations for JP/KO. (DE/FR fortunately have both effects appearing elsewhere in the repository, so I can cheat using that, although Fool's Tumble will still need translation.) Should I simply add an empty string value for each effect property where I don't know the translation, or take a chance on machine translation, or just leave it blank and open a new issue for others who know those languages?

Reply:
Yeah, that's exactly it.  If you don't have a translation, just put a FIXME there.

Fool's Tumble is 387 (https://xivapi.com/Status/387?pretty=true)
Imp is 613, 1103, 1134, 1356
Toad is 439, 441, 1101

Hopefully all of those Imp and Toad translations are the same? I'm not sure why the game needs so many.  I have a local version of SaintCoinach and just look in Status.csv to find these, and then look up the id in xivapi.

Reply:
Ahh, you're right, I totally skipped over checking XIVAPI. That makes it easy. I'll get those worked out tonight.